tests/db_access/prepared_statements.cpp

```diff
@@ -600,11 +600,17 @@ go_bandit([](){
             AssertThat(varbinary.getStringView(), Equals("ij\0kl"s));
         });
 
-        it("can work with psReadValues (throws multiple rows error)", [&](){
+        it("can work with psReadValues (zero rows error)", [&](){
```

Comment:
Shoudln't it say something like psReadValues can detect wrong result? (here it means zero result).
Actually there should probably be `describe section` for `psReadValues`. And `"zero rows error"` should be one of its tests.